mindsdb/api/mongo/functions/__init__.py

```diff
@@ -0,0 +1,9 @@
+def is_true(val):
+    return bool(val) is True
+
+
+def is_false(val):
+    return bool(val) is False
+
+
+__all__ = ['is_true', 'is_false']
```

Comment:
Is it possible to use explicit imports here ? asking because `__all__` is another python magic that not everyone may know and until now we haven't used in anywhere.

Reply:
yes, in this place it unnecessary. Removed